### DISCUSSION TITLE
Can't upgrade to Plus.

### DISCUSSION THREAD

**Discussion by braveheartwilliam** [cssbattle/forum#351] [Bugs reports]
Trying to upgrade to plus but "Upgrade ...." button not active.

**Comment by aarthvaen** [comment]
My question will be generic in its nature, but:

1. Have you tried using another browser to see if you don't get anything either?
2. Do you have an adblocker extension enabled on your browser? This can possibly block the purchase popup that you get once you click "Upgrade".

**Comment by chinchang** [comment]
This means Paddle's (our payments tool) script isn't loading in your browser. Which is most probably due to an adblocker. Would recommend turning off the adblocker for the website and then try.
Let us know if this still doesn't work.
